ngraph/test/frontend/tensorflow/tf_utils.hpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+
+static const std::string TF_FE = "tensorflow";
```

Comment:
need to change to "tf" to be aligned with frontend name

Reply:
Also tests are failed due to "tensorflow" frontend is not found